From b60ecb76a3a77a9b1cc88f8fb2c77e7dbc42d318 Mon Sep 17 00:00:00 2001
From: Fang Hui <b31070@freescale.com>
Date: Tue, 18 Nov 2014 15:44:07 +0800
Subject: [PATCH 415/602] MA-5848 - [Android_MX6SX_SD]USB Audio: It can't
 record audio from USB Microphone.100%

Add support for AUDIO_DEVICE_IN_DGTL_DOCK_HEADSET

Signed-off-by: Fang Hui <b31070@freescale.com>
---
 evk_6sl/audio_policy.conf        | 2 +-
 hdmidongle_6dq/audio_policy.conf | 2 +-
 imx53_smd/audio_policy.conf      | 2 +-
 sabreauto_6q/audio_policy.conf   | 2 +-
 sabreauto_6sx/audio_policy.conf  | 2 +-
 sabresd_6dq/audio_policy.conf    | 2 +-
 sabresd_6sx/audio_policy.conf    | 2 +-
 7 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/evk_6sl/audio_policy.conf b/evk_6sl/audio_policy.conf
index 60f7e4b..1d93299 100644
--- a/evk_6sl/audio_policy.conf
+++ b/evk_6sl/audio_policy.conf
@@ -42,7 +42,7 @@ audio_hw_modules {
         sampling_rates 8000|11025|16000|22050|24000|32000|44100|48000
         channel_masks AUDIO_CHANNEL_IN_MONO|AUDIO_CHANNEL_IN_STEREO
         formats AUDIO_FORMAT_PCM_16_BIT
-        devices AUDIO_DEVICE_IN_BUILTIN_MIC|AUDIO_DEVICE_IN_WIRED_HEADSET|AUDIO_DEVICE_IN_USB_DEVICE
+        devices AUDIO_DEVICE_IN_BUILTIN_MIC|AUDIO_DEVICE_IN_WIRED_HEADSET|AUDIO_DEVICE_IN_USB_DEVICE|AUDIO_DEVICE_IN_DGTL_DOCK_HEADSET
       }
     }
   }
diff --git a/hdmidongle_6dq/audio_policy.conf b/hdmidongle_6dq/audio_policy.conf
index 60f7e4b..1d93299 100644
--- a/hdmidongle_6dq/audio_policy.conf
+++ b/hdmidongle_6dq/audio_policy.conf
@@ -42,7 +42,7 @@ audio_hw_modules {
         sampling_rates 8000|11025|16000|22050|24000|32000|44100|48000
         channel_masks AUDIO_CHANNEL_IN_MONO|AUDIO_CHANNEL_IN_STEREO
         formats AUDIO_FORMAT_PCM_16_BIT
-        devices AUDIO_DEVICE_IN_BUILTIN_MIC|AUDIO_DEVICE_IN_WIRED_HEADSET|AUDIO_DEVICE_IN_USB_DEVICE
+        devices AUDIO_DEVICE_IN_BUILTIN_MIC|AUDIO_DEVICE_IN_WIRED_HEADSET|AUDIO_DEVICE_IN_USB_DEVICE|AUDIO_DEVICE_IN_DGTL_DOCK_HEADSET
       }
     }
   }
diff --git a/imx53_smd/audio_policy.conf b/imx53_smd/audio_policy.conf
index dde94de..c35b922 100644
--- a/imx53_smd/audio_policy.conf
+++ b/imx53_smd/audio_policy.conf
@@ -42,7 +42,7 @@ audio_hw_modules {
         sampling_rates 8000|11025|16000|22050|24000|32000|44100|48000
         channel_masks AUDIO_CHANNEL_IN_MONO|AUDIO_CHANNEL_IN_STEREO
         formats AUDIO_FORMAT_PCM_16_BIT
-        devices AUDIO_DEVICE_IN_BUILTIN_MIC|AUDIO_DEVICE_IN_WIRED_HEADSET|AUDIO_DEVICE_IN_BACK_MIC|AUDIO_DEVICE_IN_USB_MIC
+        devices AUDIO_DEVICE_IN_BUILTIN_MIC|AUDIO_DEVICE_IN_WIRED_HEADSET|AUDIO_DEVICE_IN_BACK_MIC|AUDIO_DEVICE_IN_USB_MIC|AUDIO_DEVICE_IN_DGTL_DOCK_HEADSET
       }
     }
   }
diff --git a/sabreauto_6q/audio_policy.conf b/sabreauto_6q/audio_policy.conf
index 65e5f5b..635b975 100644
--- a/sabreauto_6q/audio_policy.conf
+++ b/sabreauto_6q/audio_policy.conf
@@ -49,7 +49,7 @@ audio_hw_modules {
         sampling_rates 8000|11025|16000|22050|24000|32000|44100|48000
         channel_masks AUDIO_CHANNEL_IN_MONO|AUDIO_CHANNEL_IN_STEREO
         formats AUDIO_FORMAT_PCM_16_BIT
-        devices AUDIO_DEVICE_IN_BUILTIN_MIC|AUDIO_DEVICE_IN_WIRED_HEADSET|AUDIO_DEVICE_IN_USB_DEVICE|AUDIO_DEVICE_IN_AUX_DIGITAL
+        devices AUDIO_DEVICE_IN_BUILTIN_MIC|AUDIO_DEVICE_IN_WIRED_HEADSET|AUDIO_DEVICE_IN_USB_DEVICE|AUDIO_DEVICE_IN_AUX_DIGITAL|AUDIO_DEVICE_IN_DGTL_DOCK_HEADSET
       }
     }
   }
diff --git a/sabreauto_6sx/audio_policy.conf b/sabreauto_6sx/audio_policy.conf
index 65e5f5b..635b975 100644
--- a/sabreauto_6sx/audio_policy.conf
+++ b/sabreauto_6sx/audio_policy.conf
@@ -49,7 +49,7 @@ audio_hw_modules {
         sampling_rates 8000|11025|16000|22050|24000|32000|44100|48000
         channel_masks AUDIO_CHANNEL_IN_MONO|AUDIO_CHANNEL_IN_STEREO
         formats AUDIO_FORMAT_PCM_16_BIT
-        devices AUDIO_DEVICE_IN_BUILTIN_MIC|AUDIO_DEVICE_IN_WIRED_HEADSET|AUDIO_DEVICE_IN_USB_DEVICE|AUDIO_DEVICE_IN_AUX_DIGITAL
+        devices AUDIO_DEVICE_IN_BUILTIN_MIC|AUDIO_DEVICE_IN_WIRED_HEADSET|AUDIO_DEVICE_IN_USB_DEVICE|AUDIO_DEVICE_IN_AUX_DIGITAL|AUDIO_DEVICE_IN_DGTL_DOCK_HEADSET
       }
     }
   }
diff --git a/sabresd_6dq/audio_policy.conf b/sabresd_6dq/audio_policy.conf
index 48a6f96..29051dc 100644
--- a/sabresd_6dq/audio_policy.conf
+++ b/sabresd_6dq/audio_policy.conf
@@ -42,7 +42,7 @@ audio_hw_modules {
         sampling_rates 8000|11025|16000|22050|24000|32000|44100|48000
         channel_masks AUDIO_CHANNEL_IN_MONO|AUDIO_CHANNEL_IN_STEREO
         formats AUDIO_FORMAT_PCM_16_BIT
-        devices AUDIO_DEVICE_IN_BUILTIN_MIC|AUDIO_DEVICE_IN_WIRED_HEADSET|AUDIO_DEVICE_IN_USB_DEVICE
+        devices AUDIO_DEVICE_IN_BUILTIN_MIC|AUDIO_DEVICE_IN_WIRED_HEADSET|AUDIO_DEVICE_IN_USB_DEVICE|AUDIO_DEVICE_IN_DGTL_DOCK_HEADSET
       }
     }
   }
diff --git a/sabresd_6sx/audio_policy.conf b/sabresd_6sx/audio_policy.conf
index 48a6f96..29051dc 100644
--- a/sabresd_6sx/audio_policy.conf
+++ b/sabresd_6sx/audio_policy.conf
@@ -42,7 +42,7 @@ audio_hw_modules {
         sampling_rates 8000|11025|16000|22050|24000|32000|44100|48000
         channel_masks AUDIO_CHANNEL_IN_MONO|AUDIO_CHANNEL_IN_STEREO
         formats AUDIO_FORMAT_PCM_16_BIT
-        devices AUDIO_DEVICE_IN_BUILTIN_MIC|AUDIO_DEVICE_IN_WIRED_HEADSET|AUDIO_DEVICE_IN_USB_DEVICE
+        devices AUDIO_DEVICE_IN_BUILTIN_MIC|AUDIO_DEVICE_IN_WIRED_HEADSET|AUDIO_DEVICE_IN_USB_DEVICE|AUDIO_DEVICE_IN_DGTL_DOCK_HEADSET
       }
     }
   }
-- 
1.8.0

